Buildings/Experimental/DHC/EnergyTransferStations/Combined/Generation5/Controls/Borefield.mo

```diff
@@ -0,0 +1,202 @@
+  connect(yValIso_actual, multiMax1.u)
+    annotation (Line(points={{-120,-40},{-92,-40}}, color={0,0,127}));
+  connect(u,enaSup.u)
+    annotation (Line(points={{-120,60},{-80,60},{-80,-20},{-52,-20}},color={0,0,127}));
+  connect(enaSup.y,enaBor.u1)
+    annotation (Line(points={{-28,-20},{-20,-20},{-20,-30},{-12,-30}},color={255,0,255}));
+  connect(opeVal.y,enaBor.u2)
+    annotation (Line(points={{-28,-48},{-20,-48},{-20,-38},{-12,-38}},
+                                                  color={255,0,255}));
+  connect(enaBor.y,conMix.trigger)
+    annotation (Line(points={{12,-30},{20,-30},{20,-96},{-6,-96},{-6,-92}},color={255,0,255}));
+  connect(maxTBorWatEnt.y,conMix.u_s)
+    annotation (Line(points={{-28,-80},{-12,-80}},color={0,0,127}));
+  connect(runBor.y,yPum)
+    annotation (Line(points={{92,60},{120,60}},color={0,0,127}));
+  connect(TBorWatEnt,conMix.u_m)
+    annotation (Line(points={{-120,-100},{0,-100},{0,-92}},                    color={0,0,127}));
+  connect(enaBor.y,runBor.u2)
+    annotation (Line(points={{12,-30},{20,-30},{20,20},{64,20},{64,60},{68,60}},  color={255,0,255}));
+  connect(mapSpe.y,runBor.u1)
+    annotation (Line(points={{42,80},{60,80},{60,68},{68,68}},color={0,0,127}));
+  connect(u,mapSpe.u)
+    annotation (Line(points={{-120,60},{0,60},{0,80},{18,80}},color={0,0,127}));
+  connect(speMin.y,mapSpe.f1)
+    annotation (Line(points={{-18,84},{18,84}},                  color={0,0,127}));
+  connect(min1.y,yValMix)
+    annotation (Line(points={{92,-60},{120,-60}},color={0,0,127}));
+  connect(conMix.y,min1.u2)
+    annotation (Line(points={{12,-80},{60,-80},{60,-66},{68,-66}},color={0,0,127}));
+  connect(mapVal.y,min1.u1)
+    annotation (Line(points={{42,40},{60,40},{60,-54},{68,-54}},color={0,0,127}));
+  connect(one.y,mapSpe.x2)
+    annotation (Line(points={{2,0},{12,0},{12,76},{18,76}},color={0,0,127}));
+  connect(one.y,mapSpe.f2)
+    annotation (Line(points={{2,0},{12,0},{12,72},{18,72}},color={0,0,127}));
+  connect(one.y,mapVal.f2)
+    annotation (Line(points={{2,0},{12,0},{12,32},{18,32}},color={0,0,127}));
+  connect(zer.y,mapVal.x1)
+    annotation (Line(points={{-38,40},{-10,40},{-10,48},{18,48}},color={0,0,127}));
+  connect(limVal.y,mapVal.x2)
+    annotation (Line(points={{-18,20},{6,20},{6,36},{18,36}},color={0,0,127}));
+  connect(u,mapVal.u)
+    annotation (Line(points={{-120,60},{0,60},{0,40},{18,40}},color={0,0,127}));
+  connect(limVal.y,mapSpe.x1)
+    annotation (Line(points={{-18,20},{6,20},{6,88},{18,88}},color={0,0,127}));
+  connect(zer.y,mapVal.f1)
+    annotation (Line(points={{-38,40},{-10,40},{-10,44},{18,44}},color={0,0,127}));
+  connect(zer.y,runBor.u3)
+    annotation (Line(points={{-38,40},{-10,40},{-10,52},{68,52}},color={0,0,127}));
+  annotation (
+    Diagram(
+      coordinateSystem(
+        preserveAspectRatio=false,
+        extent={{-100,-120},{100,120}})),
+    defaultComponentName="con",
```

Comment:
may change the default name to `borCon`?

Reply:
I rely on the following rationale.

- Always start the name with what the component models, e.g, `val` for a valve, `sen` for a sensor, here `con` for a controller.
- This block is intended to be used (probably exclusively?) in the subsystem model `Subsystems.Borefield` where the default name `con` is enough as only one composed control block is used there. The same holds for the chiller and HX (it was not consistent for the HX so I changed the default component name `conHex` to `con` in my last commit.)